[2017-07-19 09:51:19.242] [ERROR] dateFileLog - Error: C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:31<br> &nbsp; &nbsp; &nbsp; &nbsp;parameters:[[yjDateTime.toUTC(sTime),yjDateTime.toUTC(eTime)];],<br> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; ^<br>SyntaxError: Unexpected token ;<br> &nbsp; &nbsp;at createScript (vm.js:56:10)<br> &nbsp; &nbsp;at Object.runInThisContext (vm.js:97:10)<br> &nbsp; &nbsp;at Module._compile (module.js:542:28)<br> &nbsp; &nbsp;at Object.Module._extensions..js (module.js:579:10)<br> &nbsp; &nbsp;at Module.load (module.js:487:32)<br> &nbsp; &nbsp;at tryModuleLoad (module.js:446:12)<br> &nbsp; &nbsp;at Function.Module._load (module.js:438:3)<br> &nbsp; &nbsp;at Module.require (module.js:497:17)<br> &nbsp; &nbsp;at require (internal/module.js:20:19)<br> &nbsp; &nbsp;at buildData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:230:21)

    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-19 09:51:53.539] [ERROR] dateFileLog - Error: TypeError: date.getTime is not a function<br> &nbsp; &nbsp;at Object.toUTC (C:\NB\yujiang.Foil.Node\src\client\js\yjDateTime.js:149:24)<br> &nbsp; &nbsp;at module.exports (C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:31:33)<br> &nbsp; &nbsp;at buildData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:256:6)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:198:5<br> &nbsp; &nbsp;at Object.module.exports.read (C:\NB\yujiang.Foil.Node\src\yjMVC.cache.js:80:4)<br> &nbsp; &nbsp;at loadData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:192:15)<br> &nbsp; &nbsp;at rrnmv (C:\NB\yujiang.Foil.Node\src\yjMVC.js:188:3)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:290:4<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (C:\NB\node_modules\express\lib\router\layer.js:95:5)<br> &nbsp; &nbsp;at next (C:\NB\node_modules\express\lib\router\route.js:131:13)

    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-19 09:51:55.054] [ERROR] dateFileLog - Error: TypeError: date.getTime is not a function<br> &nbsp; &nbsp;at Object.toUTC (C:\NB\yujiang.Foil.Node\src\client\js\yjDateTime.js:149:24)<br> &nbsp; &nbsp;at module.exports (C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:31:33)<br> &nbsp; &nbsp;at buildData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:256:6)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:198:5<br> &nbsp; &nbsp;at Object.module.exports.read (C:\NB\yujiang.Foil.Node\src\yjMVC.cache.js:80:4)<br> &nbsp; &nbsp;at loadData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:192:15)<br> &nbsp; &nbsp;at rrnmv (C:\NB\yujiang.Foil.Node\src\yjMVC.js:188:3)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:290:4<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (C:\NB\node_modules\express\lib\router\layer.js:95:5)<br> &nbsp; &nbsp;at next (C:\NB\node_modules\express\lib\router\route.js:131:13)

    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-19 09:52:09.326] [ERROR] dateFileLog - Error: TypeError: date.getTime is not a function<br> &nbsp; &nbsp;at Object.toUTC (C:\NB\yujiang.Foil.Node\src\client\js\yjDateTime.js:149:24)<br> &nbsp; &nbsp;at module.exports (C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:31:33)<br> &nbsp; &nbsp;at buildData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:256:6)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:198:5<br> &nbsp; &nbsp;at Object.module.exports.read (C:\NB\yujiang.Foil.Node\src\yjMVC.cache.js:80:4)<br> &nbsp; &nbsp;at loadData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:192:15)<br> &nbsp; &nbsp;at rrnmv (C:\NB\yujiang.Foil.Node\src\yjMVC.js:188:3)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:290:4<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (C:\NB\node_modules\express\lib\router\layer.js:95:5)<br> &nbsp; &nbsp;at next (C:\NB\node_modules\express\lib\router\route.js:131:13)

    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-19 09:52:10.746] [ERROR] dateFileLog - Error: TypeError: date.getTime is not a function<br> &nbsp; &nbsp;at Object.toUTC (C:\NB\yujiang.Foil.Node\src\client\js\yjDateTime.js:149:24)<br> &nbsp; &nbsp;at module.exports (C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:31:33)<br> &nbsp; &nbsp;at buildData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:256:6)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:198:5<br> &nbsp; &nbsp;at Object.module.exports.read (C:\NB\yujiang.Foil.Node\src\yjMVC.cache.js:80:4)<br> &nbsp; &nbsp;at loadData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:192:15)<br> &nbsp; &nbsp;at rrnmv (C:\NB\yujiang.Foil.Node\src\yjMVC.js:188:3)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:290:4<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (C:\NB\node_modules\express\lib\router\layer.js:95:5)<br> &nbsp; &nbsp;at next (C:\NB\node_modules\express\lib\router\route.js:131:13)

    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-19 09:53:01.139] [ERROR] dateFileLog - Error: TypeError: date.getTime is not a function<br> &nbsp; &nbsp;at Object.toUTC (C:\NB\yujiang.Foil.Node\src\client\js\yjDateTime.js:149:24)<br> &nbsp; &nbsp;at module.exports (C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:31:32)<br> &nbsp; &nbsp;at buildData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:256:6)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:198:5<br> &nbsp; &nbsp;at Object.module.exports.read (C:\NB\yujiang.Foil.Node\src\yjMVC.cache.js:80:4)<br> &nbsp; &nbsp;at loadData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:192:15)<br> &nbsp; &nbsp;at rrnmv (C:\NB\yujiang.Foil.Node\src\yjMVC.js:188:3)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:290:4<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (C:\NB\node_modules\express\lib\router\layer.js:95:5)<br> &nbsp; &nbsp;at next (C:\NB\node_modules\express\lib\router\route.js:131:13)

    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-19 09:53:02.435] [ERROR] dateFileLog - Error: TypeError: date.getTime is not a function<br> &nbsp; &nbsp;at Object.toUTC (C:\NB\yujiang.Foil.Node\src\client\js\yjDateTime.js:149:24)<br> &nbsp; &nbsp;at module.exports (C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:31:32)<br> &nbsp; &nbsp;at buildData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:256:6)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:198:5<br> &nbsp; &nbsp;at Object.module.exports.read (C:\NB\yujiang.Foil.Node\src\yjMVC.cache.js:80:4)<br> &nbsp; &nbsp;at loadData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:192:15)<br> &nbsp; &nbsp;at rrnmv (C:\NB\yujiang.Foil.Node\src\yjMVC.js:188:3)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:290:4<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (C:\NB\node_modules\express\lib\router\layer.js:95:5)<br> &nbsp; &nbsp;at next (C:\NB\node_modules\express\lib\router\route.js:131:13)

    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-19 09:57:41.079] [ERROR] dateFileLog - Error: ReferenceError: yjDateTime is not defined<br> &nbsp; &nbsp;at paramIniTenMinutes (C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:50:17)<br> &nbsp; &nbsp;at module.exports (C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:28:15)<br> &nbsp; &nbsp;at buildData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:256:6)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:198:5<br> &nbsp; &nbsp;at Object.module.exports.read (C:\NB\yujiang.Foil.Node\src\yjMVC.cache.js:80:4)<br> &nbsp; &nbsp;at loadData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:192:15)<br> &nbsp; &nbsp;at rrnmv (C:\NB\yujiang.Foil.Node\src\yjMVC.js:188:3)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:290:4<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (C:\NB\node_modules\express\lib\router\layer.js:95:5)<br> &nbsp; &nbsp;at next (C:\NB\node_modules\express\lib\router\route.js:131:13)

    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-19 09:58:47.400] [ERROR] dateFileLog - Error: ReferenceError: yjDateTime is not defined<br> &nbsp; &nbsp;at paramIniTenMinutes (C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:50:17)<br> &nbsp; &nbsp;at module.exports (C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:28:15)<br> &nbsp; &nbsp;at buildData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:256:6)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:198:5<br> &nbsp; &nbsp;at Object.module.exports.read (C:\NB\yujiang.Foil.Node\src\yjMVC.cache.js:80:4)<br> &nbsp; &nbsp;at loadData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:192:15)<br> &nbsp; &nbsp;at rrnmv (C:\NB\yujiang.Foil.Node\src\yjMVC.js:188:3)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:290:4<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (C:\NB\node_modules\express\lib\router\layer.js:95:5)<br> &nbsp; &nbsp;at next (C:\NB\node_modules\express\lib\router\route.js:131:13)

    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-19 09:58:49.506] [ERROR] dateFileLog - Error: ReferenceError: yjDateTime is not defined<br> &nbsp; &nbsp;at paramIniTenMinutes (C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:50:17)<br> &nbsp; &nbsp;at module.exports (C:\NB\tm.FGEMS\bizServer\biz\demand\getFeebackUsage.{post}.{m}.js:28:15)<br> &nbsp; &nbsp;at buildData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:256:6)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:198:5<br> &nbsp; &nbsp;at Object.module.exports.read (C:\NB\yujiang.Foil.Node\src\yjMVC.cache.js:80:4)<br> &nbsp; &nbsp;at loadData (C:\NB\yujiang.Foil.Node\src\yjMVC.js:192:15)<br> &nbsp; &nbsp;at rrnmv (C:\NB\yujiang.Foil.Node\src\yjMVC.js:188:3)<br> &nbsp; &nbsp;at C:\NB\yujiang.Foil.Node\src\yjMVC.js:290:4<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (C:\NB\node_modules\express\lib\router\layer.js:95:5)<br> &nbsp; &nbsp;at next (C:\NB\node_modules\express\lib\router\route.js:131:13)

    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-19 10:15:09.801] [ERROR] dateFileLog - { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'GROUP BY   UNIX_TIMESTAMP(ddh.SatTime) DIV 60*15' at line 1
    at Query.Sequence._packetToError (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (C:\NB\node_modules\mysql\lib\Connection.js:82:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (C:\NB\node_modules\mysql\lib\Connection.js:185:25)
    at doQuery (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:128:25)
    at success (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:119:13)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:57:11
    at Ping.onPing (C:\NB\node_modules\mysql\lib\Pool.js:94:5)
    at bound (domain.js:280:14)
    at Ping.runBound [as _callback] (domain.js:293:12)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlState: '42000',
  index: 0 }
[2017-07-19 10:15:09.829] [ERROR] dateFileLog - Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'GROUP BY   UNIX_TIMESTAMP(ddh.SatTime) DIV 60*15' at line 1
    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-19 10:15:13.861] [ERROR] dateFileLog - { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'GROUP BY   UNIX_TIMESTAMP(ddh.SatTime) DIV 60*15' at line 1
    at Query.Sequence._packetToError (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (C:\NB\node_modules\mysql\lib\Connection.js:82:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (C:\NB\node_modules\mysql\lib\Connection.js:185:25)
    at doQuery (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:128:25)
    at success (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:119:13)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:57:11
    at Ping.onPing (C:\NB\node_modules\mysql\lib\Pool.js:94:5)
    at bound (domain.js:280:14)
    at Ping.runBound [as _callback] (domain.js:293:12)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlState: '42000',
  index: 0 }
[2017-07-19 10:15:13.869] [ERROR] dateFileLog - Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'GROUP BY   UNIX_TIMESTAMP(ddh.SatTime) DIV 60*15' at line 1
    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-19 11:20:36.257] [ERROR] dateFileLog - C:\NB\tm.FGEMS\bizServer\biz\auto\simFeeback.js:11
    rule.minute = [  48,49,50,51,52,53,54,55,56,57,58,59, 40, 30,21,22,23,24,25,26,27,28 20,29 ,,11,12,13,14, 10, 1,2,3,4,5,6,7,8,9,0];  
                                                                                         ^^
SyntaxError: Unexpected number
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.foundFile (C:\NB\yujiang.Foil.Node\src\yjApp.js:192:7)
[2017-07-19 11:20:47.536] [ERROR] dateFileLog - C:\NB\tm.FGEMS\bizServer\biz\auto\simFeeback.js:11
    rule.minute = [  48,49,50,51,52,53,54,55,56,57,58,59, 40, 30,21,22,23,24,25,26,27,28 20,29 ,11,12,13,14, 10, 1,2,3,4,5,6,7,8,9,0];  
                                                                                         ^^
SyntaxError: Unexpected number
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.foundFile (C:\NB\yujiang.Foil.Node\src\yjApp.js:192:7)
